index.html

```diff
@@ -6,13 +6,246 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>CodersCamp | Projekt 1 - HTML i CSS</title>
-    <meta name="description" content="CodersCamp to darmowy, otwarty kurs programowania webowego we Wrocławiu. Sprawdź, czego nauczysz się na CodersCamp i spróbuj swoich sił w branży IT!">
-
+    <meta name="description" content="myPorfolio">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Russo+One&display=swap" rel="stylesheet">   -->
```

Comment:
Nie powinno się zostawiać takich zakomentowanych części kodu.
Po to właśnie jest system kontroli wersji - Git. Jeśli dobrze się nim posługujesz i opisujesz commity, to wtedy z łatwością znajdziesz zmainę - jeśli będziesz chciał do czegoś wrócić. A tak to nie wiadomo po co ten kod jest i czy w ogóle będzie działa jak się go odkomentuje.